transactions/build.gradle

```diff
@@ -14,11 +14,22 @@
  * limitations under the License.
  */
 
+group = 'org.ehcache'
+
+apply plugin: EhOsgi
 apply plugin: EhDeploy
+apply plugin: EhPomMangle
+apply plugin: EhDocs
 
 dependencies {
   compile project(':impl'), project(':xml')
   compile group: 'javax.transaction', name: 'jta', version: '1.1'
   testCompile project(path: ':core-spi-test')
   compile group: 'org.codehaus.btm', name: 'btm', version: '2.1.4'
 }
+
+// For EhPomMangle
+dependencies {
+  shadow "org.ehcache:ehcache:$parent.baseVersion"
```

Comment:
Does this only work because we never link the shadow plugin? If yes, I would suggest using different names to clear up confusion, like done in `clustered/dist` with `pomOnlyProvided`